### DISCUSSION TITLE
RX sensitivity with Lime SDR mini 2.0

### DISCUSSION THREAD

**Discussion by gabtas** [srsran/srsRAN_Project#771] [General Help]
I’m characterizing the RX sensitivity of an eNodeB based on srsRAN and LimeSDR mini 2.0.
I have a RFFE which includes a SAW duplexer for UL and DL isolation. I use a reference UE (evk of LTE module) and a step attenuator in order to tune the path loss for the 95% of throughput drop.

By **comparing** the eNodeB (UL) sensitivity achieved with **LimeSDR mini 2.0** with the one achieved with **another SDR**, I can measure a **sensitivity reduction of about 10dB**. The srsRAN settings are the same (ie: Band 5 850MHz, 25RB (5MHz), rx_gain=40) and the TX power level (DL) is adjusted at the same level.
The NF estimation is around 15dB which is much higher than the 2dB of the LMS7002M transceiver.
I’m wondering if there is a particular setting in srsRAN enb.conf file I have to apply in the case of LimeSDR mini 2.0. Maybe the default IF filtering is not optimized for Lime?

Thanks in advance for your support

**Comment by herlesupreeth** [comment]
This repo is for 5G RAN not LTE (eNB). Please raise the issue in srsRAN_4G repo

**Reply by gabtas** [reply]
I'm sorry, thanks
